sdk/python/kfp_tekton/tekton.py

```diff
@@ -177,3 +180,68 @@ def CEL_ConditionOp(condition_statement):
     ConditionOp = ConditionOp_template(condition_statement)
     ConditionOp.add_pod_annotation("valid_container", "false")
     return ConditionOp
+
+
+def Break():
+    '''A BreakOp template for Break Operation using PipelineLoop
+    '''
+    BreakOp_yaml = '''\
+    name: 'pipelineloop-break-operation'
+    description: 'Break Operation using PipelineLoop'
+    implementation:
+        container:
+            image: %s
+            command:
+            - sh
+            - -c
+            - |
+              echo "$0"
+            args:
+            - "break loop"
+    ''' % (TEKTON_BASH_STEP_IMAGE)
+    BreakOp_template = components.load_component_from_text(BreakOp_yaml)
+    BreakOp = BreakOp_template()
+    return BreakOp
+
```

Comment:
To be honest I did not follow how the break happens here. Seems it's just a normal task?

Reply:
@pugangxa I think it's related to issue #800.

Reply:
Yes, right now the pipeline loop break only works when the `last-loop-task` is skipped. However, the proposed `Break()` DSL is to break when the condition is met. So I opened #800 to add a new break features to break when the `Break()` task is being executed.

Reply:
Sorry I just overlooked that issue. Then it makes sense. Thanks for all your explanation.